manual/manual/cmds/top.etex

```diff
@@ -126,6 +126,10 @@ The following command-line options are recognized by the "ocaml" command.
 \begin{unix}
 The following environment variables are also consulted:
 \begin{options}
+\item["OCAMLTOP_UTF_8"] When printing string values, non-ascii bytes
+(>0x7E) are printed as decimal escape sequence if "OCAMLTOP_UTF_8" is
```

Comment:
`(>0x7E)` will format poorly in LaTeX, with the `>` character rendered as upside-down question mark or some such.  Please format as a proper math formula:
```
  ($ {} > "\0x7E" $)
```

Reply:
Fixed.